lib/user/getPlayerInfo.js

```diff
@@ -0,0 +1,42 @@
+// Includes
+var http = require('../util/http.js').func;
+var parser = require('cheerio');
+
+// Args
+exports.required = ['userId'];
+
+// Define
+exports.func = function (args) {
+  return http({
+    url: '//www.roblox.com/users/' + args.userId + '/profile',
+    options: {
+      resolveWithFullResponse: true,
+      followRedirect: false
+    }
+  })
+  .then(function (res) {
+    if (res.statusCode === 200) {
+      let body = parser.load(res.body);
+
+      let blurb = body('.profile-about-content-text').text();
+      let status = body('div[data-statustext]').attr('data-statustext');
+      let username = body('.header-title h2').text();
+      let joinDate = body('.profile-stats-container .text-lead').text().slice(0,9);
+
+      joinDate = Date.parse(joinDate);
```

Comment:
For consistency throughout the project please use [rbx.getDate](https://github.com/sentanos/roblox-js/blob/master/lib/util/getDate.js), which accepts a time zone (and handles daylight savings), and use ROBLOX's main time zone (which is central time). Although this isn't an exact timestamp and it's hard to check which time zone it actually is (ROBLOX can be inconsistent too), the best bet is to just use the same thing as everything else.